server/ai_worker_test.go

```diff
@@ -29,11 +28,6 @@ import (
 
 func TestRemoteAIWorker_Error(t *testing.T) {
 	httpc := &http.Client{Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}}
-	//test request
```

Comment:
@ad-astra-video I'm probablly overlooking something but this appears to be unused.

Reply:
Believe that is correct.  Moved creating jobs to `createAIJob` and forgot to delete this out.

Reply:
No problem 👍🏻.